apps/cyberstorm-remix/app/styles/layout.css

```
@@ -15,284 +20,443 @@
+    /* Discrete width: the stack snaps to a standard rail width — 120/160/180/300
+       (+ the slot's 1px borders) — the largest that fits the gutter, chosen by
+       the nimbus-ad-gutter @container rules below. Always <= the gutter so it
+       can't overflow the page; createAd is then given only the sizes that fit
+       this box (see fittingSizes in nitroAds.ts). */
+    width: calc(120px + 2px);
+
+    /* Fill the viewport height minus the header, the top offset above and a
+       gap below, AND the floating video's reserve, so no rail ad sits behind the
+       docked player. */
+    height: calc(
+      100dvh - var(--header-height) - var(--gap-md) - var(--gap-2xs) -
```

Stylelint (17.14.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)